llvm/lib/Target/PowerPC/PPCRegisterInfo.cpp

```diff
@@ -1102,13 +1102,20 @@ void PPCRegisterInfo::lowerCRBitSpilling(MachineBasicBlock::iterator II,
     SpillsKnownBit = true;
     break;
   default:
+    // When spilling a CR bit, The super register may not be explicitly defined
```

Comment:
```suggestion
    // When spilling a CR bit, the super register may not be explicitly defined
```
Minor nit on the comment.

Reply:
Fixed, thanks.